third_party/terraform/website/docs/r/composer_environment.html.markdown

```diff
@@ -167,7 +167,7 @@ The `config` block supports:
 The `node_config` block supports:
 
 * `zone` -
-  (Optional)
+  (Required)
   The Compute Engine zone in which to deploy the VMs running the
   Apache Airflow software, specified as the zone name or
   relative resource name (e.g. "projects/{project}/zones/{zone}"). Must belong to the enclosing environment's project 
```

Comment:
There are also some lines below (see L176 and L187) that need to be changed. I'm not really sure to what, because I basically copied what the [composer docs](https://cloud.google.com/composer/docs/reference/rest/v1/projects.locations.environments#nodeconfig) say. We should probably bug them to fix their docs or their validation.